R/helpers.R

```diff
@@ -68,6 +74,8 @@ use_dependency <- function(package, type, min_version = NULL) {
   invisible(TRUE)
 }
 
+r_version <- function() glue("{R.Version()$major}.{R.Version()$minor}")
```

Comment:
This will produce an R version with 3 components, whereas I think we want to default to 2 components. I.e. we want `4.1` instead of `4.1.2`. Despite the name `minor` appears to return `minor.patch`.

Reply:
Oh, good catch. Thanks! Relatedly, do you think we embed CRAN's advice of only specifying to the minor level by inserting a check when the user supplies `min_version`, something like `check_r_version <- function(min_version) if length(strsplit(min_version, "\\.")[[1]]) != 2 {...}`?

Reply:
I think if someone provides the full version number, we assume they know what they're doing.